webapp/src/components/ServerError.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import ErrorPage from './ErrorPage'
+
+export default function ServerError() {
+  return (
+    <ErrorPage>
+      <h2>
+        Uh-oh. <br />
+        We couldn’t retrieve information from the server.
+      </h2>
+      <div className="error-message">
+        <p>Please try again in a few minutes.</p>
+        <br/>
+        <p>
+          If the problem persists, contact us at our&nbsp;
+          <a href="https://chat.decentraland.org" target="_blank" rel="noopener noreferrer">
+            Community Chat
```

Comment:
I like the idea of using `Community Chat` instead of `Rocket Chat` but the other error pages still say the latter.